src/api/index.js

```diff
@@ -1,8 +1,10 @@
+import { AMBROSUS_TOKEN_URL } from '../config';
+
 export const ambPriceInUsd = async (amb) => {
   try {
-    const ambPrice = await fetch('https://token.ambrosus.io')
+    const ambPrice = await fetch(AMBROSUS_TOKEN_URL)
       .then((response) => response.json())
-      .then((data) => data?.data?.price_usd);
+      .then((data) => data.data.price_usd);
```

Comment:
you can use destructuring assignment in arguments, like
`( { data: { price_usd } } ) => price_usd`

Reply:
Looks better, but the linter swears(